package.json

```diff
@@ -40,11 +40,9 @@
     "lint:types": "tsc --noEmit",
     "prepack": "pnpm build",
     "preexample": "vite build",
-    "test": "vitest",
-    "test:all": "concurrently \"pnpm:test:run\" \"pnpm:test:example\" --prefix-colors auto",
+    "test": "concurrently \"pnpm:test:*\" --names \"test:\" --prefix-colors auto",
     "test:example": "pnpm preexample && cd examples && pnpm test",
-    "test:run": "vitest run",
-    "test:ui": "vitest --ui"
+    "test:vitest": "vitest run --exclude dist-for-testing --exclude tests"
```

Comment:
I renamed the folder `tests` (previously used to store files for `vitest`), because `@codemod-utils/tests` expects fixtures and test files for its testing to be in the `tests` folder.